benchmark/checks/run_benchmark_messages_mem.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+DATE=`LC_ALL=C date +"%d.%m.%Y_%H:%M"`
+echo "Performance report for $DATE"
+echo -e "-----------------------------\n"
+
+AGENTS=10
+NUM_RUNS=10
```

Comment:
Number runs should be higher to get more reliable measures. I'd say 100

Reply:
will update a bit later